### ISSUE TITLE
No depthwise convolution in ShuffleNet?

### ISSUE THREAD

**Issue by bareblackfoot** [kuangliu/pytorch-cifar#70]
There is no depthwise convolution in the shufflenet code :(

**Comment by kuangliu** [comment]
`nn.Conv2d(...groups=in_channels...)` is depthwise convolution.
